src/uwtools/file.py

```diff
@@ -126,4 +161,36 @@ def go(self):
         """
         linkname = lambda k: Path(self._target_dir / k if self._target_dir else k)
         yield "File links"
-        yield [symlink(target=Path(v), linkname=linkname(k)) for k, v in self._file_map.items()]
+        yield [symlink(target=Path(v), linkname=linkname(k)) for k, v in self._config.items()]
+
+
+class MkDir(Stager):
```

Comment:
I don't like this name, but haven't thought of anything better. `Copier` and `Linker` are concise and describe what they do, and are nouns. `MkDir` is (maybe) a verb, so that's a mismatch. Something like `DirectoryMaker` would align better, but is kind of painful. I think the `mkdir` term will be familiar to many users, so may have some merit. I'm not really sure.

Reply:
It makes it a little overloaded, but I'm at a loss for something better

Reply:
Yes, I think that's better, will do. This seems like a good case to adopt the Python idiom instead of the Unix one.

Reply:
If we made this `MakeDirs`, would we also want to see
```
makedirs:
  - d1
  - d2
```
in YAML and `uw file makedirs` -- i.e. consistency across the board here, even though it's longer?

And `makedir` or `makedirs` if so?

Reply:
I made the `mkdir` -> `makedirs` and `MkDir` -> `MakeDirs` changes. They're easy to revert if we end up not liking them.